mtz-wizard.html

```diff
@@ -168,13 +165,10 @@
        *
        * @param {HTMLElement[]} steps
        * @param {String} attr - the attrForSelected property
-       * @return {String}
        */
-      __computeFallbackSelection(steps, attr) {
-        let fallback = '';
-        if (attr && steps && steps.length > 0)
-          fallback = steps[0].getAttribute(attr);
-        return fallback;
+      __setDefaultSelection(steps, attr) {
+        if (!this.selected && attr && steps && steps.length > 0)
```

Comment:
eslint should be complaining about this, either single line it (if it's under 120 and you're fine with that) or wrap it in {} if it bleeds to multiple lines @stramel pointed this out to me